hub/hub-api.yaml

```
@@ -2364,6 +2492,42 @@ components:
           type: string
           description: The Inperium app.
 
+    FeatureUsageRequest:
+      type: object
+      description: | 
+        The incoming object sent by Inperium services to Hub when a feature usage is updated outside the Hub. 
+        For example, you make internaltional call (in Inperium Talk) and the feature usage must be updated.
+      properties:
+        quantity:
+          type: integer
+          format: int64
+          minimum: 0
+          description: | 
+            The quantity of consumed feature. It can be either the total amount of feature usage or its increment. 
+            It depends on the end-point that is in use.
+
+    FeatureUsage:
+      type: object
+      description: The PlanFeature object describes the feature in the plan you have including consumed quantity, its limitation, etc. 
+      properties:
+        plan:
+          $ref: "#/components/schemas/Plan"
+        feature:
+          $ref: "#/components/schemas/Feature"
+        priceModel:
+          type: string
+        isInfinite:
```

```suggestion
        infinite:
```
I think we try to stick to the idea of "booleans without prefixes", because they are quite understandable

---

Thanks Maxim. I'll keep it in mind. For now, the attribute is removed anyway.